Firestore/core/src/local/persistence.h

```diff
@@ -118,6 +119,8 @@ class Persistence {
   virtual DocumentOverlayCache* GetDocumentOverlayCache(
       const credentials::User& user) = 0;
 
+  virtual OverlayMigrationManager* GetOverlayMigrationManager() = 0;
```

Comment:
Add method comment.
`  /** Returns the migration manager responsible for calculating and saving overlays. */` ?